.github/ISSUE_TEMPLATE/release.md

```diff
@@ -10,7 +10,7 @@ assignees: ArangoGutierrez, marquiz, zvonkok
 <!--
 Please do not remove items from the checklist
 -->
-- [ ] All [OWNERS](https://github.com/kubernetes-sigs/node-feature-discovery-operator/blob/master/OWNERS) must LGTM the release proposal
+- [ ] All [OWNERS](https://sigs.k8s.io/node-feature-discovery-operator/blob/master/OWNERS) must LGTM the release proposal
```

Comment:
Hmm, I think the subdirs don't work this way

This would work but I think it is probably best to stick with the real github URLs for other than go package naming

```
- [ ] All [OWNERS](https://sigs.k8s.io/node-feature-discovery-operator/OWNERS) must LGTM the release proposal
```